test/e2e/deprecated_e2e_test.go

```diff
@@ -39,8 +55,8 @@ var _ = Describe("Not found APIs", func() {
 				name: "installplan contains a missing API",
 				IP: &operatorsv1alpha1.InstallPlan{
 					ObjectMeta: metav1.ObjectMeta{
-						Namespace: *namespace, // this is necessary due to ginkgo table semantics, see https://github.com/onsi/ginkgo/issues/378
-						Name:      "test-plan-api",
+						// Namespace: ns.GetName(), // this is necessary due to ginkgo table semantics, see https://github.com/onsi/ginkgo/issues/378 - setting namespace in describe table
```

Comment:
There is just one table entry.  It may better not use the tableEntries to avoid this.